samples/python/visual_language_chat/video_to_text_chat.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+import argparse
+import numpy as np
+import cv2
+import openvino_genai
+from openvino import Tensor
+from pathlib import Path
+
+
+def streamer(subword: str) -> bool:
+    '''
+
+    Args:
+        subword: sub-word of the generated text.
+
+    Returns: Return flag corresponds whether generation should be stopped.
+
+    '''
+    print(subword, end='', flush=True)
+
+    # No value is returned as in this example we don't want to stop the generation in this method.
+    # "return None" will be treated the same as "return openvino_genai.StreamingStatus.RUNNING".
+
+
+def read_video(path: str, num_frames: int = 8) -> Tensor:
+    '''
+
+    Args:
+        path: The path to the video.
+        num_frames: Number of frames sampled from the video.
+
+    Returns: the ov.Tensor containing the video.
+
+    '''
+    cap = cv2.VideoCapture(path)
+
+    frames = []
+
+    while cap.isOpened():
+        ret, frame = cap.read()
+        if not ret:
+            break
+
+        frames.append(np.array(frame))
```

Comment:
Align C++ and Python: Pythons temporary keeps all frames while C++ immediately discards some

Reply:
Aligned Python sample with C++ one.